mmv1/products/bigqueryconnection/api.yaml

```diff
@@ -183,6 +192,9 @@ objects:
           - !ruby/object:Api::Type::Boolean
             name: 'useParallelism'
             description: If parallelism should be used when reading from Cloud Spanner
+          - !ruby/object:Api::Type::Boolean
+            name: 'useServerlessAnalytics'
```

Comment:
Do you mind adding a test or updating a test to use this value? Thanks!

Reply:
sure, will do

Reply:
done